Makefile

```diff
@@ -82,3 +82,7 @@ bootstrap: bats ksonnet-lib
 
 build_and_test:
 	./script/start-test-environment.sh "make binary && make controller-image CONTROLLER_IMAGE=bitnami/kubeless-controller-manager:latest && make integration-tests"
+
+.PHONY: build-manifests
+build-manifests:
+	./script/build-manifests.sh
```

Comment:
missing blank line at the end here

Reply:
The job for [minikube_kafka-v1.18.9](https://app.circleci.com/pipelines/github/kubeless/kafka-trigger/49/workflows/90744719-a105-4f1d-a23b-923285a934be/jobs/306)  fails if `conntrack` is not installed.

> Kubernetes 1.18.9 requires conntrack to be installed in root's path

Not sure why is required though.

Reply:
Fixed